source/Energinet.DataHub.MeteringPoints.Application/Validation/ValidationErrors/MeteringPointMustNotBeKnownRuleError.cs

```diff
@@ -16,9 +16,9 @@
 
 namespace Energinet.DataHub.MeteringPoints.Application.Validation.ValidationErrors
 {
-    public class MeteringPointMustBeKnownRuleError : ValidationError
+    public class MeteringPointMustNotBeKnownRuleError : ValidationError
```

Comment:
Should be ValidationError instead of RuleError:
```suggestion
    public class MeteringPointMustNotBeKnownValidationError : ValidationError
```